src/test/java/guitests/ContextMenuTests.java

```diff
@@ -81,6 +81,22 @@ public void test2() {
     }
 
     /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_twelveIssuesInListView_markIssue7andBelowReadUnread() {
```

Comment:
This name doesn't look like it follows the convention. The third part should be the expected result?